src/Sylius/Component/Promotion/Repository/PromotionCouponRepositoryInterface.php

```diff
@@ -23,6 +23,12 @@ public function createQueryBuilderByPromotionId($promotionId): QueryBuilder;
 
     public function countByCodeLength(int $codeLength): int;
```

Comment:
I wonder whether we could merge the new method with this one (`countByCodeLength(5) === countByCodeLengthWithPrefixAndSuffix(5, null, null)`), but I'm not sure, just thinking out loud.